remix-gql-tailwind/app/components/FileExplorer/FileExplorer.view.tsx

```diff
@@ -1,9 +1,9 @@
-import type { ExplorerItem } from './types';
```

Comment:
It looks like prettier is set to single quote but these changed to double. Double check on that you might need to run prettier